src/core/server/app/middleware/passport/strategies/jwt.ts

```diff
@@ -99,7 +99,10 @@ export async function verifyAndRetrieveUser(
   } catch (err) {
     // When the JWT was revoked, just indicate that there is no user on the
     // request rather than erroring out.
-    if (err instanceof JWTRevokedError) {
+    if (
+      err instanceof JWTRevokedError ||
+      err.jse_cause instanceof TokenExpiredError
```

Comment:
Would you mind if we use `.cause()` here instead?

I suggest this because `jse_cause` and its older version `we_cause` is considered a private field and may change over time as we update node versions.

source: https://github.com/joyent/node-verror/issues/37